Guides/MongoDBSetUp.md

```diff
@@ -0,0 +1,59 @@
+This guide outlines the steps needed to setup the MongoDB Connection for storing the sensor data
+
+1. Run the following to install MongoDB on RPI
+	
+	```
+	sudo apt-get update
+	sudo apt-get upgrade
+	sudo apt-get install mongodb-server
+	```
+
+2. To make sure mongodb runs on RPI start up
+	
+	```sudo service mongodb start```
+
+
+3. To test that MongoDB has been installed successfully, run 
+
+	```mongo```
+
+	This will run the MongoDB shell where you can issue mongoDB commands
+
+
+5. Create a new database with name "solarsensereports"
+		
+		Run: 
+
+		```use solarsensereports```
+
+		after the database has been created, exit the mongodb shell by running: 
+
+		```exit```
+
+5. Install the Python MongoDB driver (PyMongo)
+
+	```python3 -m pip install pymongo==3.4.0```
+
+	Opted to use 3.4.0 because the latest (3.7.2) was having problems connecting with the latest mongo release
+
+6. Run the python daemon from folder SensorReader, sensorReader.py, and make sure there is data reported 	at least once so you can test the next step
```

Comment:
What do you mean here? I try to run `python3 sensorReader.py` but it just stays at "Connection to Blocker established 0", and nothing shows up in the mongoDB when I check the next step...
![image](https://user-images.githubusercontent.com/35539589/47609247-05f32780-d9f0-11e8-9853-82b757fc6b30.png)

Reply:
Did you run the daemon?

Reply:
I'm able to get a reading when I run `python3 /opt/miflora-mqtt-daemon/miflora-mqtt-daemon.py`, but running `python3 sensorReader.py` and `python3 miflora-mqtt-daemon.py` gives me the error: 
`Traceback (most recent call last):
  File "miflora-mqtt-daemon.py", line 129, in <module>
    reporting_mode = config['General'].get('reporting_method', 'mqtt-json')
  File "/usr/lib/python3.5/configparser.py", line 956, in __getitem__
    raise KeyError(key)
KeyError: 'General'`